src/updater.cpp

```diff
@@ -135,9 +135,9 @@ void Updater::downloadUpdate(QString link)
     form.addWidget(msgLabel);
     dialog.open();
     QProcess *process = new QProcess(this);
-    QString cmd = QString("cd $TMPDIR && curl -LO %1 && tar -xf minikube-gui-macos.tar.gz && rm "
-                          "minikube-gui-macos.tar.gz")
-                          .arg(link);
+    QString cmd = QString("cd $TMPDIR && curl -LO %1 && mkdir -p minikube-gui-macos && "
+                          "tar -xf minikube-gui-macos.tar.gz -C minikube-gui-macos && "
+                          "rm minikube-gui-macos.tar.gz").arg(link);
     process->start("bash", { "-c", cmd });
     process->waitForFinished(-1);
     QDialogButtonBox buttonBox(Qt::Horizontal, &dialog);
```

Comment:
in line 147 we can get rid of Step 1 and 2

```
   "Download complete!\n\nAfter clicking 'Restart' do the following:\n\n1. You will see: "
            "\"minikube-gui cannot be opened...\" click 'Cancel'\n2. Your Appliciations folder "
            "will be opened, right click on minikube-gui and click 'Open'")
```

and also for the Button that says restart, we can name it, "re-oepn gui " so ppl dont think it restarts actual minikube